src/Posts/Post.jsx

```diff
@@ -10,6 +10,8 @@ const showFlagAccountFeature = props.showFlagAccountFeature;
 
 State.init({
   hasBeenFlagged: false,
+  showFlaggedToast: false,
```

Comment:
Separate boolean to control Toast so that closing the toast doesn't reshow the flagged post.